modules/velostrata-manager-vm/main.tf

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+/******************************************
+Velostrata VM IP Address
+ *****************************************/
+
+resource "google_compute_address" "velo-frontend-ip" {
+  project = var.project_id
+  region  = trimsuffix(var.velostrata_vm_zone, regex("-[^-]+$", var.velostrata_vm_zone))
+  name    = "velostrata-frontend-ip"
+}
+
+
+/******************************************
+Velostrata VM Instance
+ *****************************************/
+
+resource "google_compute_instance" "velo-vm" {
```

Comment:
Could we use https://github.com/terraform-google-modules/terraform-google-vm/tree/master/modules/compute_instance ?

Reply:
sounds good, I will test it out :)